lib/ops/ops.go

```diff
@@ -455,6 +456,14 @@ type Certificates interface {
 	DeleteClusterCertificate(SiteKey) error
 }
 
+// EnvironmentVariables manages runtime environment variables in cluster
+type EnvironmentVariables interface {
+	// CreateUpdateEnvarsOperation creates a new operation to update cluster runtime environment variables
+	CreateUpdateEnvarsOperation(CreateUpdateEnvarsOperationRequest) (*SiteOperationKey, error)
```

Comment:
I would either expand `Envars` to `EnvironmentVariables` here, or vice versa in the method below, for naming consistency.